be/src/exec/vectorized/hash_join_node.cpp

```diff
@@ -350,6 +350,17 @@ bool HashJoinNode::_has_null(const ColumnPtr& column) {
 Status HashJoinNode::_build(RuntimeState* state) {
     {
         SCOPED_TIMER(_build_conjunct_evaluate_timer);
+        // Currently, in order to achieve simplicity, HashJoinNode uses BigChunk,
```

Comment:
Implement is to bring about, to put into practice while achieve is to succeed in something, now especially in academic performance. Generally speaking, we prefer implement in a software project.

Reply:
Fixed